src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/code/CompressedReadStream.java

```diff
@@ -87,6 +101,12 @@ public int readInt() {
                                     CompressedReadStream::setPosition);
   }
 
+  @Override
+  public void setPosition(int position) {
+    this.position = position;
```

Comment:
Maybe a call to `super()` would be better.

Reply:
ok. thanks!